doc/guides/west/manifest.rst

```diff
@@ -768,12 +778,12 @@ The ``import`` key can also contain a mapping with the following keys:
 .. _pathlib:
    https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match
 
-Whitelists override blacklists if both are given. For example, if a project is
-blacklisted by path, then whitelisted by name, it will still be imported.
+Allowlists override blocklists if both are given. For example, if a project is
+blocklisted by path, then allowlisted by name, it will still be imported.
```

Comment:
Do those verbs actually exist? :) not sure if "blocked" and "allowed" would be better here

Reply:
Same for other instance further down.

Reply:
Ah, the perils of search and replace. Thanks.